projects/demo/src/modules/components/icons/icons.tokens.ts

```diff
@@ -2,9 +2,9 @@ import {inject} from '@angular/core';
 import type {TuiStringHandler} from '@taiga-ui/cdk';
 import {tuiCreateTokenFromFactory} from '@taiga-ui/cdk';
 import {TUI_SVG_OPTIONS} from '@taiga-ui/core';
-import * as allIcons from '@taiga-ui/icons';
+import {TUI_ALL_ICONS} from '@taiga-ui/icons';
```

Comment:
Should we still compile a typescript dictionary of all the icons? Doesn't seem like it makes sense now that you cannot pass source code to the icon component. Preloading icons in bundle would work differently now.

Reply:
I dropped it in my PR, I don't think if we maintain name equivalence we need a separate page to list all the 1500 icons.

Reply:
That's why I also want to drop file/name processing and just write it like `iconLeft="@tui.align-horizontal-distribute-start"` so you can just copy icon name from Lucide docs.